recipes/rapidjson/all/conanfile.py

```diff
@@ -14,7 +14,7 @@ class RapidjsonConan(ConanFile):
     homepage = "http://rapidjson.org"
     license = "MIT"
     package_type = "header-library"
-    package_id_embed_mode = "minor"
+    package_id_embed_mode = "minor_mode"
```

Comment:
Actually, what does it mean `minor_mode` for conan when versions are "cci" versions like rapidjson?

Reply:
For "cci" versions, major is "cci" and minor is date, e.g. "20220822"; patch is `None`. My understanding is that setting `package_id_embed_mode = "minor_mode"` (as opposed to the default `full_mode`), leads to a situation where a consuming library only will need to rebuild their binary if minor (or major) version has changed. With `full_mode`, this happens for recipe revisions as well.

Relevant conan docs:
[Package id modes](https://docs.conan.io/2.0/reference/conanfile/attributes.html#package-id-embed-non-embed-unknown-mode)
[The effect of dependencies on `package_id`](https://docs.conan.io/2.0/reference/binary_model/dependencies.html#reference-binary-model-dependencies)